### PR TITLE
Update backlog components / pattern list to latest

### PR DESCRIPTION
**Pull request by nickcolley**

### CODE REVIEW

File: src/community/backlog/index.md.njk [hunk 1]
```diff
@@ -18,13 +18,20 @@ Here is a list of the components, patterns and updates currently on the Design S
 
 
 <table class="govuk-table app-table--constrained">
-  <tbody><tr>
-    <th class="govuk-table__header">Name</th>
-    <th class="govuk-table__header" style="width: 90px; text-align: right">Status</th>
-  </tr>
-
-
+  <thead>
```
Comment (36degrees) [inline src/community/backlog/index.md.njk]: This is a good improvement :+1: Do you think it's worth splitting into a separate commit? It's not really covered by the current commit message.

File: src/community/backlog/index.md.njk [hunk 2]
```diff
@@ -366,12 +449,20 @@ Here is a list of the components, patterns and updates currently on the Design S
...
+        Proposed
+      </td>
+    </tr>
     <tr>
       <td class="govuk-table__cell">
         <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/30">Modal dialogue</a>
       </td>
       <td class="govuk-table__cell govuk-body-s" style="text-align: right">
-        In progress
+        To do
```
Comment (36degrees) [inline src/community/backlog/index.md.njk]: Is this correct?

Comment (nickcolley) [reply src/community/backlog/index.md.njk]: It's 'todo' in the projects board (these changes are from scraping the board) so if it's not the project board should be updated

File: src/community/backlog/index.md.njk (hunk 1 above)
Comment (nickcolley) [reply src/community/backlog/index.md.njk]: Split out

File: src/community/backlog/index.md.njk (hunk 2 above)
Comment (36degrees) [reply src/community/backlog/index.md.njk]: I've moved the card to 'In Progress' – is it easy to update this PR to match?

File: src/community/backlog/index.md.njk (hunk 1 above)
Comment (36degrees) [reply src/community/backlog/index.md.njk]: 👏

File: src/community/backlog/index.md.njk (hunk 2 above)
Comment (nickcolley) [reply src/community/backlog/index.md.njk]: Sure thing

**Pull request merged**